src/image_bios.hpp

```diff
@@ -16,16 +16,13 @@ struct BIOSUpdater : public FwUpdBase
 
     void reset() override
     {
-        // There is nothing to reset in the BIOS.
+        // Not supported yet.
```

Comment:
Why? Will you create a separate ticket to support it?

Reply:
Yes, I will. 
We can't just erase whole NVRAM, it contains service headers with GUIDs, CRC, etc.